jupyterlab_translate/utils.py

```diff
@@ -289,13 +291,65 @@ def get_line(lines: List[str], value: str) -> str:
     return str(line_count)
 
 
+def _extract_schema_strings(
+    schema: dict, schema_lines: list, ref_path: str, prefix: str = ""
+):
+    message_context = "schema"
+
+    to_translate = [
+        "title",
+        "description",
+        "properties/.*/title",
+        "properties/.*/description",
+        "definitions/properties/.*/title",
+        "definitions/properties/.*/description",
+        # JupyterLab-specific
+        "jupyter.lab.setting-icon-label",
+        "jupyter.lab.menus/.*/label"
+        # TODO: toolbar?
+        # TODO: add custom paths specified in jupyter.lab.internationalization
+    ]
```

Comment:
Assuming the following structure:
```ts
interface "jupyter.lab.internationalization": {
  "domain": string,
  "selectors": string[]
}
```
```suggestion
    ]
    to_translate += schema.get("jupyter.lab.internationalization", {}).get("selectors", [])
```